receivers/alertmanager/testing.go

```diff
@@ -0,0 +1,13 @@
+package alertmanager
+
+// FullValidConfigForTesting a string representation of JSON object that contains all fields supported by the notifier Config. Can be used without secrets
```

Comment:
```suggestion
// FullValidConfigForTesting is a string representation of JSON object that contains all fields supported by the notifier Config. It can be used without secrets.
```

Reply:
The same applies to every other instance of this comment.